### ISSUE TITLE
Estimated extent query failed on view

### ISSUE THREAD

**Issue by zachsa** [CrunchyData/pg_tileserv#156]
I'm running `pg_tileserv` from the `pramsey/pg_tileserv:20220815` image.

After creating a view that could potentially return a large number of rows, I get an error in the logs if I try to go the JSON file:

```
https://pg_tileserv.domain/public.my_new_view.json

> Estimated extent query failed, run 'ANALYZE public.my_new_view'
```

However, if I try to run `ANALYZE public.my_new_view` I get back the message:

```
skipping "my_new_view" --- cannot analyze non-tables or special system tables
```

I have also tried restarting the `pg_tileserv` service. But that doesn't help either

**Comment by pramsey** [comment]
Yeah, estimated extent on views is going to be problematic for sure. Really nothing to do except skip them...
